ui/raidboss/data/07-dt/hunts/ss_rank.ts

```diff
@@ -0,0 +1,143 @@
+// import Outputs from '../../../../../resources/outputs';
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// TODO: Add minions?
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'HuntDTSS',
+  zoneId: [
+    ZoneId.Urqopacha,
+    ZoneId.Kozamauka,
+    ZoneId.YakTel,
+    ZoneId.Shaaloani,
+    ZoneId.HeritageFound,
+    ZoneId.LivingMemory,
+  ],
+  zoneLabel: {
+    en: 'SS Rank Hunts',
+    de: 'SS Jagdziele',
+    fr: 'Objectifs de chasse SS',
+    ja: 'SSモブ',
+    cn: 'SS 级狩猎怪',
+    ko: 'SS급 마물',
+  },
+  triggers: [
+    {
+      id: 'Hunt Arch Aethereater Aetherodynamics',
+      type: 'StartsUsing',
+      // Not clear why there are four ids that can be used -- might be connected to the Soulless Stream variants?
+      netRegex: {
+        id: ['9A57', '9A58', '9B9F', '9BA0'],
+        source: 'Arch Aethereater',
+        capture: false,
+      },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Hunt Arch Aethereater Obliterate',
+      type: 'StartsUsing',
+      netRegex: { id: '9A5B', source: 'Arch Aethereater' },
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'Hunt Arch Aethereater Meltdown',
+      type: 'StartsUsing',
+      netRegex: { id: '9A5C', source: 'Arch Aethereater', capture: false },
+      durationSeconds: 3, // these are fast and they happen back to back without much of a telegraph
+      response: Responses.awayFromFront(),
+    },
+    // There's some weirdness in the logs where the heat/cold debuffs may be reapplied.
+    // Total duration doesn't change, but suppress just for safety.
+    {
+      id: 'Hunt Arch Aethereater Heatstroke',
+      type: 'GainsEffect',
+      netRegex: { effectId: '102D' },
+      condition: Conditions.targetIsYou(),
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 3,
+      suppressSeconds: (_data, matches) => parseFloat(matches.duration),
+      response: Responses.stopMoving(),
+    },
+    {
+      id: 'Hunt Arch Aethereater Cold Sweats',
+      type: 'GainsEffect',
+      netRegex: { effectId: '102E' },
+      condition: Conditions.targetIsYou(),
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 3,
+      suppressSeconds: (_data, matches) => parseFloat(matches.duration),
+      infoText: (_data, _matches, output) => output.frozen!(),
+      outputStrings: {
+        frozen: {
+          en: 'Frozen soon',
```

Comment:
Maybe use https://github.com/OverlayPlugin/cactbot/blob/3c26eea707ae02aea558670b4c96ca9c46542bcb/ui/raidboss/data/06-ew/alliance/euphrosyne.ts#L330-L337 here? (slightly different mechanic but the output is nearly the same)

Reply:
I might be mis-remembering, but isn't that the mech variant the one that's like more typical blizzard mechs (as in Vali Ex/SoS Ex/etc) where you have to keep moving to avoid getting frozen?  Looking at some of the translations (like 動く), it looks like this is more indicative of an instruction to keep moving -- but for this mech, it's just a forced frozen-in-place that doesn't require movement to avoid.

Reply:
Yes, this is one of those "keep moving to not get frozen" mechs.  If the ja translation is directly stating to keep moving, then it wouldn't be appropriate here.

Reply:
In that case, it probably makes sense to have whatever call we use here be re-translated rather than just copy/pasting, in case there's linguistic nuance we're missing, being a different mech and all.  I'm fine with as-is or `(frozen soon)` -- I think 'freeze' is maybe a bit ambiguous, especially since it can be used as a verb (instructive) or a noun (descriptive).  LMK if you have a preference though.

Reply:
No preference here.